src/components/common/TmPage.vue

```diff
@@ -133,7 +133,7 @@ export default {
 
 .tm-page-main {
   position: relative;
-  min-height: 100vh;
+  /*min-height: 100vh;*/
```

Comment:
@jbibla should we just remove this? It seems using the `vh` was causing some undesired side-effects for bigger screens